File: show/src/show/cOM.java
package show;
	public class cOM {
		int object;
		int salienceFactor;

		public int read(byte[] b, int offset) {
			return offset;
		}

		public cOM(int o, int sf) {
			object = o;
			salienceFactor = sf;
		}

		public cOM(LittleEndianDataInputStream rs) {
			object = rs.readInteger();
			salienceFactor = rs.readInteger();
		}
	}

File: Web/pyLPBackEnd/EclipseWorkspace/pyLPBackEnd/src/COM.py
import struct

class COM:

    def __init__(self, *, rs=None, o=None, s=None):
        if rs is not None:
            self.object, self.salienceFactor = struct.unpack('<2i', rs.f.read(8))
        else:
            self.object = o
            self.salienceFactor = s
